lib/db/mem.js

```diff
@@ -380,6 +380,33 @@ module.exports = function (log, error) {
       )
   }
 
+  Memory.prototype.sessionDevice = function (id) {
```

Comment:
As purely a naming nit, I wonder if this might be better as something like "sessionWithDevice"; the current name makes it sound like it's returning the device record that belongs to the session, rather than returning the session with some additional data.